rustls-cert-gen/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+#![warn(missing_docs)]
+//! This library wraps [rcgen] to provide a simple API to generate TLS
+//! certificate-chains. Its primary intent is to ease development of
+//! applications that verify chain of trust. It can be used for
+//! whatever purpose you may need a TLS certificate-chain.
+
+mod cert;
+pub use cert::{Ca, CertificateBuilder, EndEntity};
```

Comment:
I think we want `EndEntityParams` and `CaParams` here too (or whatever they end up named if you agree with my rename suggestion).

Right now the return types from `CertificateBuilder::end_entity` and `CertificateBuilder::ca` are opaque out of crate.